android/app/src/googleplay/java/org/fossasia/openevent/utils/ClusterRenderer.java

```diff
@@ -0,0 +1,38 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.maps.android.clustering.ClusterManager;
+import com.google.maps.android.clustering.view.DefaultClusterRenderer;
+
+import org.fossasia.openevent.R;
+
+/**
+ * Created by rohanagarwal94 on 4/7/17.
+ */
+public class ClusterRenderer extends DefaultClusterRenderer<MicrolocationClusterWrapper> {
```

Comment:
The point is not about the build flavor, it's just about the architecture of your components. You have put them in utils where they are only used in 1 class. They are not utils for the whole app. They should be either along with fragment they are created for, or in a separate package. 

google play module is not a package, it's a build flavor which gets merged in the main module when the app is built